js/main.js

```diff
@@ -4,16 +4,31 @@ var $username = document.getElementById('username');
 var $fullName = document.getElementById('fullName');
 var $location = document.getElementById('location');
 var $bio = document.getElementById('bio');
-var $profilePicture = document.querySelector('img');
+var $profilePicture = document.getElementById('profilePicture');
 var $form = document.querySelector('form');
 var $profile = document.getElementById('profile');
 var $viewNodeList = document.querySelectorAll('.view');
 var $header = document.querySelector('header');
+var $entries = document.getElementById('entries');
+var $createEntry = document.getElementById('create-entry');
+var $imgUrl = document.getElementById('imgUrl');
+var $entryPicture = document.getElementById('entryPicture');
+var $entryForm = document.getElementById('entryForm');
+var $entryTitle = document.getElementById('entryTitle');
+var $entryNotes = document.getElementById('entryNotes');
+var phImage = 'images/placeholder-image-square.jpg'
+var allImages = document.querySelectorAll('img');
+allImages.onerror = function(){this.src=phImage}
```

Comment:
indentation in your function.

Reply:
![image](https://user-images.githubusercontent.com/73401059/101226683-2c153800-364a-11eb-9041-025a183b04bd.png)